### PR TITLE
🩹 restart ongoing auctions if the sequencer went down

### PR DESCRIPTION
**Pull request by Thomas-Smets**
Can only be merged after https://github.com/arcadia-finance/accounts-v2/pull/175 is merged in accounts-v2

### CODE REVIEW

File: src/Liquidator.sol [hunk 1]
```diff
@@ -41,15 +42,18 @@ contract Liquidator is Owned, ReentrancyGuard, ILiquidator {
     // The base of the auction price curve (decreasing power function).
     // Determines in what time the auction price halves, 18 decimals precision.
     uint64 internal base;
-    // The time after which the auction is considered not successful, in seconds.
+    // The time after which the auction is considered not failed and can be bought in, in seconds.
```
Comment (j-vp) [inline src/Liquidator.sol:45]:
```suggestion
    // The time after which the auction is considered failed and can be bought in, in seconds.
```

File: src/Liquidator.sol [hunk 2]
```diff
@@ -59,8 +63,8 @@ contract Liquidator is Owned, ReentrancyGuard, ILiquidator {
         uint128 startDebt;
         // The base of the auction price curve.
         uint64 base;
-        // The timestamp after which the auction is considered not successful.
-        uint32 cutoffTimeStamp;
+        // The time after which the auction is considered not failed and can be bought in, in seconds.
```
Comment (j-vp) [inline src/Liquidator.sol:66]:
```suggestion
        // The time after which the auction is considered failed and can be bought in, in seconds.
```

File: src/Liquidator.sol (hunk 1 above)
Comment (j-vp) [inline src/Liquidator.sol:45]:
```suggestion
    // The time duration after which the auction is considered failed and can be bought in, in seconds.
```

+ emphasis on duration and not an absolute timestamp

**Review by acollette — APPROVED** [review]
lgtm

File: src/Liquidator.sol [hunk 3]
```diff
@@ -441,29 +507,30 @@ contract Liquidator is Owned, ReentrancyGuard, ILiquidator {
         address creditor = auctionInformation_.creditor;
```
Comment (j-vp) [inline src/Liquidator.sol:507]: L488-491: re-order the scenario checks

**Review by j-vp — APPROVED** [review]
gtg after comments

**Pull request merged**